examples/getting-started-session-based/03-serving-session-based-model-torch-backend.ipynb

```diff
@@ -0,0 +1,1011 @@
+{
```

Comment:
<p>Should we mention somehting why Torch backend? You mention in the notebook it is the recommended way to scale and have optimized inference models. I think we should mention it in the beginning</p><p> </p><p>We can mention in the end, that we have the python backend example as well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>